invenio_admin/ext.py

```diff
@@ -48,7 +59,7 @@ def __init__(self, app=None, **kwargs):
             self.init_app(app, **kwargs)
 
     def init_app(self, app, entry_point_group='invenio_admin.views',
-                 **kwargs):
+                 permission=admin_access_permission, **kwargs):
```

Comment:
what about setting the permission loader/factory class instead of an instance? Then each admin view can specify need(s) (e.g. action(s)).